doc/source/user/configoptions.rst

```diff
@@ -709,6 +709,15 @@ Networking options
       no longer cached. This can help when dealing with resources that can be
       modified during execution of GDAL-related code.
 
+-  .. config:: GDAL_HTTP_PATH_AS_IT
```

Comment:
May I suggest 
```suggestion
-  .. config:: GDAL_HTTP_PATH_AS_IT_IS
```
or maybe GDAL_HTTP_PATH_VERBATIM

Reply:
changed to GDAL_HTTP_PATH_VERBATIM